### PR TITLE
Feature/neov 295 chrome 74 tests

### PR DESCRIPTION
**Pull request by megheaiulian**
Investigating chrome 74 tests...

### CODE REVIEW

File: cosmoz-omnitable-column-date-mixin.html [hunk 1]
```diff
@@ -73,17 +73,31 @@
...
+
+			// convert value to Date
+			let date = value;
+			if (!(date instanceof Date)) {
+				// if the value is an ISO string, make sure that it has an explicit timezone
+				if (typeof value === 'string') {
+					date = this.getAbsoluteISOString(date);
+				}
+
+				date = new Date(date);
```
Comment (nomego) [inline cosmoz-omnitable-column-date-mixin.html]: Is this if date would be a number?

File: cosmoz-omnitable-column-date-mixin.html [hunk 2]
```diff
@@ -73,17 +73,31 @@
 			if (value == null || value === '') {
 				return;
 			}
-			const date = value instanceof Date ? value : new Date(value);
+
+			// convert value to Date
+			let date = value;
+			if (!(date instanceof Date)) {
+				// if the value is an ISO string, make sure that it has an explicit timezone
+				if (typeof value === 'string') {
```
Comment (nomego) [inline cosmoz-omnitable-column-date-mixin.html]: Please add some comment about Safari and timezone handling here, especially if it becomes uniform so we can remove the workaround.

File: cosmoz-omnitable-column-time.html [hunk 3]
```diff
@@ -142,7 +142,8 @@ <h3 style="margin: 0;">[[ title ]]</h3>
 				if (value == null) {
 					return;
 				}
-				value = this.toValue(this._fixedDate + 'T' + value);
+				value = this.getAbsoluteISOString(this._fixedDate + 'T' + value);
```
Comment (nomego) [inline cosmoz-omnitable-column-time.html]: no need to put on two lines?
```suggestion
				value = this.toValue(this.getAbsoluteISOString(this._fixedDate + 'T' + value));
```

File: demo/ajax-demo.html [hunk 4]
```diff
@@ -74,26 +74,25 @@ <h3>Cosmoz omnitable demo</h3>
 	</template>
```
Comment (nomego) [inline demo/ajax-demo.html]: Reformatting this file shouldn't be part of the PR, and if you touch on code like this, make sure to ES6-update it.

File: demo/custom-template-demo.html [hunk 5]
```diff
@@ -87,65 +87,63 @@ <h3>Cosmoz omnitable templates demo</h3>
 	</x-page>
 
 	<script type="text/javascript">
-/*global document, window, Cosmoz, Polymer */
```
Comment (nomego) [inline demo/custom-template-demo.html]: Same as ajax-demo

File: test/range-date.html [hunk 6]
```diff
@@ -161,14 +160,14 @@
 					max: '2023-03-21'
 				};
 				Polymer.flush();
-				assert.equal(column.filter.min.getTime(), new Date('2015-03-18T00:00:00').getTime());
```
Comment (nomego) [inline test/range-date.html]: Why are these tests updated?

File: cosmoz-omnitable-column-date-mixin.html (hunk 2 above)
Comment (megheaiulian) [reply cosmoz-omnitable-column-date-mixin.html]: Added comments and TODO.

File: test/range-date.html (hunk 6 above)
Comment (megheaiulian) [reply test/range-date.html]: Updated because we don't want to actually test date parsing in the assert expectation.

File: demo/ajax-demo.html (hunk 4 above)
Comment (megheaiulian) [reply demo/ajax-demo.html]: We'll do it in another PR.
